TAScheduler/views.py

```diff
@@ -42,7 +42,8 @@ def get(self, request):
         courses, sections = CourseManagement.load(request.user)
         print(courses)
         print(sections)
-        return render(request, "coursemanagement.html", {"Courses": courses, "Sections": sections})
+        print(request.user.groups.all()[0].name)
+        return render(request, "coursemanagement.html", {"Courses": courses, "Sections": sections, "Role":request.user.groups.all()[0].name})
```

Comment:
Instead of using `request.user.groups.all()[0].name`, why not just pass in the result of `determineRole(request.user)`?

Reply:
Also probably a better idea